lib/ruby_lsp/requests/support/common.rb

```diff
@@ -86,7 +86,7 @@ def self_receiver?(node)
           params(
             title: String,
             entries: T.any(T::Array[RubyIndexer::Entry], RubyIndexer::Entry),
-          ).returns(Interface::MarkupContent)
+          ).returns(String)
```

Comment:
Can you please double-check if we use this method anywhere else? I kind of expected stuff to break when we changed the type here.

If I'm not mistaken we use it for documentation somewhere. Since it now returns a `String` instead of an `Interface::MarkupContent`, we need to do the same thing we did in hover and instantiate it.

```ruby
Interface::MarkupContent.new(
  kind: "markdown",
  value: markdown_from_index_entries(...),
)
```

Reply:
Great point — see #1328, where I've modified `signature_help.rb` and `completion.rb` as you've described.